indy_common/config.py

```diff
@@ -97,7 +97,7 @@
 # Top level package to be updated via pool upgrade command
 UPGRADE_ENTRY = 'indy-node'
 
-PACKAGES_TO_HOLD = ['indy-plenum', 'indy-node', 'python3-indy-crypto', 'libindy-crypto',
+PACKAGES_TO_HOLD = ['indy-plenum', 'indy-node',
```

Comment:
Should we also hold Ursa?

Reply:
strictly speaking, ursa is a plenum dependency, not a node dependency, so if there is a way to do it in plenum, that's where we should do it.

Reply:
All I know is I removed those two packages because they are not needed.
I did not add ursa here because I was trusting that a test would fail if it was necessary to do so. Is that faulty logic?